### DISCUSSION TITLE
v2.7.6

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#947] [General]
## 🐛 Bug fixes

- Fix non win32 binary paths @timmo001 (#944)

## 🧰 Maintenance

- Bump package version @timmo001 (#945)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/v2.7.6'>v2.7.6</a>.</em>

**Comment by spitz66** [comment]
Thanks, this version seems to solve majority of issues at least for me. :)
I can access settings and data again and also exit works as expected.

However, CPU Load still seems to be a bit high although not as bad in 2.5.0 with a 100% peak every 30s:
![SB_4](https://user-images.githubusercontent.com/65921206/136691712-48816b0d-f908-4e36-9647-61aed23e9d74.JPG)

Every time Node Js service is rising in CPU Load, also Antimalware service follows.

Below is a graph when System Bridge is running and when exiting program:
![SB_5](https://user-images.githubusercontent.com/65921206/136692302-181c3761-4cfc-426b-ba2e-0952352594df.JPG)
